config/defaults.go

```diff
@@ -80,6 +86,11 @@ func (d *Defaults) Validate() error {
 		}
 	}
 
+	if d.KillAfter == "" {
+		fmt.Println("KillAfter time is not set-up default to 15s")
+		d.KillAfter = "15s"
+	}
+
```

Comment:
I think we should make KillAfter a required field instead of giving it a default time. Leave it here just want to get more opinion.